Print_A_Bot/djwifi/iw_parse.py

```diff
@@ -0,0 +1,300 @@
+import re
```

Comment:
this is a rather long list of methods that act as a read only "api" to the wifi manager

couple high level items

`ssid` is the wifi network name
`iwlist` is a command to list details about all wifi networks in the area